test/suites/wdio.conf.js

```diff
@@ -7,18 +7,15 @@
 
```

Comment:
Could you please try to move this file one level up while you are at it? It just makes more sense in my opinion.

Reply:
Moved!